lib/OpenQA/WebAPI.pm

```diff
@@ -325,8 +324,8 @@ sub startup {
     # job_set_waiting, job_set_continue
     my $command_r = $job_r->route('/set_:command', command => [qw(waiting running)]);
     $command_r->post('/')->name('apiv1_set_command')->to('job#set_command');
-    $job_r->post('/restart')->name('apiv1_restart')->to('job#restart');     # job_restart
-    $job_r->post('/cancel')->name('apiv1_cancel')->to('job#cancel');        # job_cancel
+    $job_r->post('/restart')->name('apiv1_restart')->to('job#restart');            # job_restart
```

Comment:
just remove these nonsense comments - they were referencing to the old (one to rule them all) Scheduler.pm. These comments change whitespace with every change and don't add any value.